hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-applicationhistoryservice/pom.xml

```diff
@@ -72,6 +72,7 @@
     <dependency>
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
+      <scope>${transient.protobuf2.scope}</scope>
```

Comment:
Any pointers why are we putting scope in every declarition rather than just putting it up in the parent pom?
https://github.com/apache/hadoop/commit/c373d3fa39013e8a8f6a6122c3ca230b4aa10abe

Reply:
1. The dependent poms (hadoop-common...) aren't publishing it any more, just using it for compilation.
2. I don't trust the base declaration to do the right thing.